drainer/sync/util.go

```diff
@@ -35,6 +35,7 @@ type DBConfig struct {
 	Checkpoint              CheckpointConfig `toml:"checkpoint" json:"checkpoint"`
 	BinlogFileDir           string           `toml:"dir" json:"dir"`
 	BinlogFileRetentionTime int              `toml:"retention-time" json:"retention-time"`
+	TiDBTxnMode             string           `toml:"tidb-txn-mode" json:"tidb-txn-mode"`
```

Comment:
how about adding  `param`(map) for arbitrary params?
(I am ok keeping `TiDBTxnMode`)